modules/formulize/include/formdisplay.php

```diff
@@ -4318,7 +4318,9 @@ function callCheckCondition(name) {
 function captureDataAsInDOM(data) {
 	jQuery('#conditionalHTMLCapture').empty();
 	jQuery('#conditionalHTMLCapture').append(data);
-	return window.document.getElementById('conditionalHTMLCapture').innerHTML.trim();
+	var capturedHTML = window.document.getElementById('conditionalHTMLCapture').innerHTML.trim();
```

Comment:
Moving forward I would recommend shying away from the use of `var` in javascript. Whenever possible use `const`, followed by `let` if mutation is required. The primary purpose is to prevent JS variable scoping issues as a best practice. All browsers have supported const/let for a long time now